samples/rtl/fladd.py

```diff
@@ -0,0 +1,241 @@
+read_a_entry, read_a_exit = state.add_map(
+    'read_a_map', dict(i='0:N//VECLEN'), schedule=dace.ScheduleType.FPGA_Device)
+read_b_entry, read_b_exit = state.add_map(
+    'read_b_map', dict(i='0:N//VECLEN'), schedule=dace.ScheduleType.FPGA_Device)
+write_c_entry, write_c_exit = state.add_map(
+    'write_c_map',
+    dict(i='0:N//VECLEN'),
+    schedule=dace.ScheduleType.FPGA_Device)
+
+# add read_a memlets and access nodes
+read_a_inp = state.add_read('fpga_A')
+read_a_out = state.add_write('A_stream')
+state.add_memlet_path(read_a_inp,
+                      read_a_entry,
+                      read_a,
+                      dst_conn='inp',
+                      memlet=dace.Memlet.simple('fpga_A', 'i'))
+state.add_memlet_path(read_a,
+                      read_a_exit,
+                      read_a_out,
+                      src_conn='out',
+                      memlet=dace.Memlet.simple('A_stream', '0'))
+
+read_b_inp = state.add_read('fpga_B')
+read_b_out = state.add_write('B_stream')
+state.add_memlet_path(read_b_inp,
+                      read_b_entry,
+                      read_b,
+                      dst_conn='inp',
+                      memlet=dace.Memlet.simple('fpga_B', 'i'))
+state.add_memlet_path(read_b,
+                      read_b_exit,
+                      read_b_out,
+                      src_conn='out',
+                      memlet=dace.Memlet.simple('B_stream', '0'))
+
+# add tasklet memlets
+A = state.add_read('A_stream')
+B = state.add_read('B_stream')
+C = state.add_write('C_stream')
+state.add_memlet_path(A,
+                      rtl_tasklet,
+                      dst_conn='a',
+                      memlet=dace.Memlet.simple('A_stream', '0'))
+state.add_memlet_path(B,
+                      rtl_tasklet,
+                      dst_conn='b',
+                      memlet=dace.Memlet.simple('B_stream', '0'))
+state.add_memlet_path(rtl_tasklet,
+                      C,
+                      src_conn='c',
+                      memlet=dace.Memlet.simple('C_stream', '0'))
+
+# add write_c memlets and access nodes
+write_c_inp = state.add_read('C_stream')
+write_c_out = state.add_write('fpga_C')
+state.add_memlet_path(write_c_inp,
+                      write_c_entry,
+                      write_c,
+                      dst_conn='inp',
+                      memlet=dace.Memlet.simple('C_stream', '0'))
+state.add_memlet_path(write_c,
+                      write_c_exit,
+                      write_c_out,
+                      src_conn='out',
+                      memlet=dace.Memlet.simple('fpga_C', 'i'))
+
+# add copy to device state
+copy_to_device = sdfg.add_state('copy_to_device')
+cpu_a = copy_to_device.add_read('A')
+cpu_b = copy_to_device.add_read('B')
+dev_a = copy_to_device.add_write('fpga_A')
+dev_b = copy_to_device.add_write('fpga_B')
+copy_to_device.add_memlet_path(cpu_a,
+                               dev_a,
+                               memlet=dace.Memlet.simple('A', '0:N//VECLEN'))
```

Comment:
`Memlet.simple` has been deprecated, please update to use the regular `Memlet` constructor, e.g.:
```python
dace.Memlet("A[0:N//VECLEN]")
```

Reply:
Fixed in https://github.com/carljohnsen/dace/commit/556c5a6deab8dc3fb020d9d09720e21021ca76ed